packages/components/src/components/editor/condition-builder/ConditionGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import { IvyIcons } from '@axonivy/ui-icons';
+import { Condition, type ConditionProps } from './Condition';
+import { useConditionBuilderContext, type LogicalOperator } from './ConditionBuilderContext';
+import { BasicSelect } from '@/components/common/select/select';
+import { Flex } from '@/components/common/flex/flex';
+import { Label } from '@radix-ui/react-label';
+import { Button } from '@/components/common/button/button';
+
+export interface ConditionGroup {
+  conditions: Condition[];
+  logicalOperator: LogicalOperator;
+}
+
+interface ConditionGroupProps extends Pick<ConditionProps, 'groupIndex'> {
+  group: ConditionGroup;
+  groupCount: number;
+}
+
+export const ConditionGroup = ({ group, groupIndex, groupCount }: ConditionGroupProps) => {
+  const { updateLogicalOperator, addCondition, removeConditionGroup, conditionMode, logicalOperatorOptions } = useConditionBuilderContext();
+
+  return (
+    <Flex direction='column' gap={2} className='condition-builder__group'>
```

Comment:
I assume you have this class here also for the impls so they can test this more easily?
I started here to streamline this additional class names a bit (see the other components). 
They always start with `ui-` it think, so in this case I would name them something like `ui-condition-builder-group`